app/views/shared/_navbar.html.erb

```diff
@@ -11,7 +11,24 @@
     <%= form.submit "Search", class:"btn-gold btn-default max-content" %>
   <% end %>
 
-  <div class="dropdown">
-    <%= image_tag "avatar.png", id: "dropdown" %>
+  <%# Dropdown avatar %>
+  <div class="dropdown" data-controller="dropdown-menu">
+    <button type="button" data-dropdown-menu-target="button" data-action="click->dropdown-menu#toggleMenu" class="btn-drop" aria-haspopup="true" aria-expanded="true">
+    </button>
+
+    <div class="menu-options hidden" data-dropdown-menu-target="menu">
+      <div>
+        <div class="options" role="menu" aria-orientation="vertical">
+          <%= link_to "See all pieces of art", arts_path, role: "menuitem", class: "option" %>
+          <button type="submit" class="btn-gold" role="menuitem">
+            <% if current_user %>
+              <%= link_to "log out", destroy_user_session_path %>
+            <% else %>
+              <%= link_to "log in", user_session_path %>
+            <% end %>
+          </button>
```

Comment:
There's no reason to have a link inside a button
Remove the button element and just keep the if else with the links inside.
The you can give the btn-default class and the btn-gold class to both link_to
to log_out use turbo-method: :delete in the link_to